lukasz-lapczyk/recruitment-web-app
# Add interests number to admin panel

This closes #13 

# Overview
Admin panel displays a number of interests in the system of "health" type, with names starting with "cosm" and chosen by women aged between 20 and 30.

# Testing prerequisites
- db seeds

# Testing
- navigate to root directory, log in as admin user
- navigate to admin panel
- [ ] admin panel should display 'Number of interests: 2' message
